x-template.js

```diff
@@ -638,8 +522,6 @@ class TemplateEngine {
   }
 
   static #canReuseDom(preparedResult, rawResult) {
-    // TODO: Is it possible that we might have the same strings from a different
```

Comment:
Because we no longer support svg, we can delete this TODO. There are no other languages than html and so this check is perfectly sufficient (without qualification).